resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/submit/LoggingPodStatusWatcher.scala

```diff
@@ -42,17 +44,24 @@ private[k8s] class LoggingPodStatusWatcherImpl(conf: KubernetesDriverConf)
 
   private var podCompleted = false
 
+  private var resourceToOldReceived = false
```

Comment:
This looks like a typo. Do you want `resourceTooOldReceived` instead of `resourceToOldReceived`?